query_test.go

```diff
@@ -1151,20 +1151,20 @@ func TestQueryReadEmpty(t *testing.T) {
 	// Submit query
 	assert.Nil(t, query.Submit())
 
-	// Validate status, since query was used this is should be complete
+	// Validate status, since array was empty this is should be incomplete
 	status, err := query.Status()
 	assert.Nil(t, err)
-	assert.Equal(t, TILEDB_COMPLETED, status)
+	assert.Equal(t, TILEDB_INCOMPLETE, status)
```

Comment:
@Shelnutt2 this needs to be incomplete for test to pass, has the behavior changed in the case of reading an empty array?

Reply:
No, behavior change in core. Incomplete is an error here. I think it might be the size of the buffers, try increasing them slightly and see. With such small buffer sometimes the estimation in core about if it can run a query or not is off from what is expected.